Fix to read access violation in StockpileIterator::operator++ (#3160)
DFHack/dfhack: docs/changelog.txt

- `hotkeys`: hotkey hints on menu popup will no longer get their last character cut off by the scrollbar
- ``launchdf``: launch Dwarf Fortress via the Steam client so Steam Workshop is functional
- `blueprint`: interpret saplings, shrubs, and twigs as floors instead of walls

> - ``Buildings::StockpileIterator:``: fix to use world map boundaries when stockpile coordinates exist outside of the world map. Fixes unhandled exception for `combine`.

Reviewer: ```suggestion
- `combine`: fix error processing stockpiles with boundaries that extend outside of the map
```
